routing/routing.go

```diff
@@ -179,6 +179,10 @@ type LBEndpoint struct {
 	// Detected represents the time when skipper instances first detected a new LB endpoint. This detection
 	// time is used for the fade-in feature of the round-robin and random LB algorithms.
 	Detected time.Time
+
+	// There is ongoing effort to move data from Metrics and Detected to this field to simplify addition of new metrics
+	// and to make them host-wide instead of route-wide
+	MetricsFromRegistry atomic.Value // Metrics
```

Comment:
This field is:
+ stored by endpointregistry postprocessor
+ loaded by fadein postprocessos
+ will be loaded by loadbalancer

I am not sure whether it is store and load at concurently (even when stored value is equal to the current value), so I have decided to make this field atomic.

Reply:
Lets add this field in a separate PR. It also does not have to be atomic.Value because it is only read by proxy

Reply:
I cannot add this in separate PR, because it seems we would like this comment to be resolved by this PR.
https://github.com/zalando/skipper/pull/2795#discussion_r1427804501

Reply:
As far as I will be able to resolve without this field, I am able to add it in the separate PR.
https://github.com/zalando/skipper/pull/2795#discussion_r1428097342